boards/nucleo-f767zi/include/periph_conf.h

```diff
@@ -153,6 +153,56 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          (sizeof(uart_config) / sizeof(uart_config[0]))
 /** @} */
 
+/**
+ * @name    SPI configuration
+ *
+ * @note    The spi_divtable is auto-generated from
+ *          `cpu/stm32_common/dist/spi_divtable/spi_divtable.c`
+ * @{
+ */
+static const uint8_t spi_divtable[2][5] = {
+    {       /* for APB1 @ 30000000Hz */
+        7,  /* -> 117187Hz */
+        5,  /* -> 468750Hz */
+        4,  /* -> 937500Hz */
+        2,  /* -> 3750000Hz */
+        1   /* -> 7500000Hz */
+    },
+    {       /* for APB2 @ 60000000Hz */
+        7,  /* -> 234375Hz */
+        6,  /* -> 468750Hz */
+        5,  /* -> 937500Hz */
+        3,  /* -> 3750000Hz */
+        2   /* -> 7500000Hz */
+    }
+};
+
+static const spi_conf_t spi_config[] = {
+    {
+        .dev      = SPI1,
+        .mosi_pin = GPIO_PIN(PORT_A, 7),
+        .miso_pin = GPIO_PIN(PORT_A, 6),
+        .sclk_pin = GPIO_PIN(PORT_A, 5),
+        .cs_pin   = GPIO_PIN(PORT_A, 4),
```

Comment:
To be compatible with Arduino pinout, this should be PD14 (D10) or just leave it as GPIO_UNDEF and let the developer set it via a custom driver configuration.